examples/interchain_queries_client_test.go

```diff
@@ -0,0 +1,304 @@
+	require.NotEmpty(t, gaiaAddress)
+
+	conns, err := r.GetConnections(ctx, eRep, stride.Config().ChainID)
+	require.NoError(t, err)
+
+	chans, err := r.GetChannels(ctx, eRep, stride.Config().ChainID)
+	require.NoError(t, err)
+
+	atomDenomTrace := transfertypes.ParseDenomTrace(transfertypes.GetPrefixedDenom(chans[0].PortID, chans[0].ChannelID, gaia.Config().Denom))
+
+	registerZoneCmdGaia := []string{
+		stride.Config().Bin, "tx",
+		"stakeibc", "register-host-zone",
+		conns[0].ID, gaia.Config().Denom, gaia.Config().Bech32Prefix,
+		atomDenomTrace.IBCDenom(), chans[0].ChannelID, "1",
+		"--chain-id", stride.Config().ChainID,
+		"--home", stride.HomeDir(),
+		"--keyring-backend", "test",
+		"--from", strideAdminAccount,
+		"--gas", "1000000",
+		"--node", stride.GetRPCAddress(),
+		"-y",
+	}
+
+	_, _, err = stride.Exec(ctx, registerZoneCmdGaia, nil)
+	require.NoError(t, err)
+
+	// Wait a few blocks for the ICA accounts to be setup
+	err = test.WaitForBlocks(ctx, 15, stride, gaia)
+	require.NoError(t, err)
+
+	gaiaVal1Address, err := gaia.(*cosmos.CosmosChain).Validators[0].KeyBech32(ctx, "validator", "val")
+	require.NoError(t, err)
+
+	gaiaVal2Address, err := gaia.(*cosmos.CosmosChain).Validators[0].KeyBech32(ctx, "validator", "val")
+	require.NoError(t, err)
+
+	addValidatorCmd := []string{
+		stride.Config().Bin, "tx",
+		"stakeibc", "add-validator",
+		gaia.Config().ChainID, "gval1", gaiaVal1Address,
+		"10", "5",
+		"--chain-id", stride.Config().ChainID,
+		"--home", stride.HomeDir(),
+		"--keyring-backend", "test",
+		"--from", strideAdminAccount,
+		"--gas", "1000000",
+		"--node", stride.GetRPCAddress(),
+		"-y",
+	}
+
+	_, _, err = stride.Exec(ctx, addValidatorCmd, nil)
+	require.NoError(t, err)
+
+	err = test.WaitForBlocks(ctx, 10, stride)
+	require.NoError(t, err)
+
+	addValidator2Cmd := []string{
+		stride.Config().Bin, "tx",
+		"stakeibc", "add-validator",
+		gaia.Config().ChainID, "gval2", gaiaVal2Address,
+		"10", "10",
+		"--chain-id", stride.Config().ChainID,
+		"--home", stride.HomeDir(),
+		"--keyring-backend", "test",
+		"--from", strideAdminAccount,
+		"--gas", "1000000",
+		"--node", stride.GetRPCAddress(),
+		"-y",
+	}
+
+	_, _, err = stride.Exec(ctx, addValidator2Cmd, nil)
+	require.NoError(t, err)
+
+	err = test.WaitForBlocks(ctx, 150, stride, gaia /*, osmosis*/)
+	require.NoError(t, err)
+
+}
+
+const (
+	dayEpochIndex    = 1
+	dayEpochLen      = "40s"
+	strideEpochIndex = 2
+	strideEpochLen   = "100s"
+	intervalLen      = 1
+	votingPeriod     = "30s"
+	maxDepositPeriod = "30s"
+)
+
+func modifyGenesisStride() func([]byte) ([]byte, error) {
+	return func(genbz []byte) ([]byte, error) {
+		g := make(map[string]interface{})
+		if err := json.Unmarshal(genbz, &g); err != nil {
+			return nil, fmt.Errorf("failed to unmarshal genesis file: %w", err)
+		}
+
+		if err := dyno.Set(g, dayEpochLen, "app_state", "epochs", "epochs", dayEpochIndex, "duration"); err != nil {
```

Comment:
til `dyno` this is a nice way to deal with this in go

Reply:
I haven't used `dyno`. It seems fine for our use case. 

I've also had success with http://jsoniter.com/migrate-from-go-std.html and https://github.com/Jeffail/gabs for other options if `dyno` ever gives us trouble.